index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML TECH DOCUMENTATION</title>
+</head>
+<body>
+ <h1>HTML DOCUMENTATION</h1>  
+ <h2>Introduction</h2> 
+ <p>HTML is an acronym for Hyper Text Markup Language and it is the standard language for creating web-pages. Every webpage you surf on your browser is built using HTML. The primary use of HTML to provide a proper structure to the web page so all the content or data of the page could represent adequately.
+<p>A stand-alone HTML can create only stastic and skeleton looking black and white pages, but with the help of CSS and JavaScript, we can create a more interactive and inuitive webpage. When we try to visit a website or click on the link, we basically request the server to show us the page, then the server acts on our request and sends us an appropriate HTML document as a response. Then this HTML document parsed by browse, and we are able to see the content.</p>
```

Comment:
#Missing Code Block
Kindly Complete the code Block that is remaining

you can use

`<code> </code>`
or
`<pre> </pre>`